package.json

```diff
@@ -15,11 +15,11 @@
     "dotenv": "^16.4.7",
     "hono": "^4.0.9",
     "openai": "^4.79.3",
-    "spinai": "*",
-    "tsx": "^4.7.1"
+    "spinai": "*"
```

Comment:
idk who needs to hear this but wildcard is sus 😬 use a fixed version for 'spinai' to avoid unexpected bugs fr.